### PR TITLE
perf: report exceptions to Sentry when configured

### PR DESCRIPTION
**Pull request by jirikuncar**
Requires `SENTRY_DSN` variable to be set and `sentry-sdk` package to be installed.

### CODE REVIEW

**Comment by rokroskar** [comment]
add docs?

File: renku/cli/move.py [hunk 1]
```diff
@@ -15,7 +15,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Move a file in the repository."""
+"""Move or rename a file, a directory, or a symlink.
+
+Moving a files that belongs to a dataset will update its metadata. It also
```
Comment (rokroskar) [inline renku/cli/move.py]:
```suggestion
Moving a file that belongs to a dataset will update its metadata. It also
```

File: renku/cli/move.py [hunk 2]
```diff
@@ -15,7 +15,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Move a file in the repository."""
+"""Move or rename a file, a directory, or a symlink.
```
Comment (rokroskar) [inline renku/cli/move.py]: this isn't relevant for a PR on sentry

Comment (jirikuncar) [reply renku/cli/move.py]: It is at least in a separate commit.

Comment (rokroskar) [reply renku/cli/move.py]: better than nothing I suppose

**Pull request merged**